src/app/(privateRoutes)/dashboard/admin/admin.module.css

```diff
@@ -0,0 +1,3 @@
+.contentAdmin {
```

Comment:
Here is a type error, change it to `adminContent`

Reply:
fixed